library/src/main/java/com/google/android/material/motion/ReactiveView.java

```diff
@@ -0,0 +1,102 @@
+package com.google.android.material.motion;
+
+/*
+ * Copyright 2016-present The Material Motion Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import android.annotation.TargetApi;
+import android.graphics.PointF;
+import android.os.Build;
+import android.view.View;
+
+import com.google.android.material.motion.properties.ViewProperties;
+
+/**
+ * A ReactiveView is a view that wraps around an Android View class.
+ * It contains Reactive Properties that you can subscribe to.
+ */
+
+public class ReactiveView {
+  View v;
+  public ReactiveView(View v) {
+    this.v = v;
+  }
+
+  public ReactiveProperty<PointF> position() {
```

Comment:
please keep this as `translation()`. There is no widely-agreed on concept of "position" in Android unfortunately